tests/modules/entrezdirect/esummary/test.yml

```
@@ -23,4 +23,4 @@
     - entrezdirect/esummary
   files:
     - path: output/entrezdirect/test_assembly.xml
-      md5sum: 01c962bc82d3b3bcb9d4da5b0b6b408a
+      md5sum: 01c962bc82d3b3bcb9d4da5b0b6b408a
```

Hopefully this will restore the final line return
```suggestion
      md5sum: 01c962bc82d3b3bcb9d4da5b0b6b408a
```